README.md

```diff
@@ -1,6 +1,27 @@
-# Project Name
+# Lego Website (Proxy Server and Docker Practice) #
+## Procedures ##
+1. GalleryComponent-master
+    - Go to "GalleryComponent-master" file and run "npm install" for the dependencies
+    - Seed the database
+        1. Make sure you add mysql config in "db" folder
+        2. Seed the "schema.sql" file using command "mysql -u root < schema.sql" in Ubuntu.
+        3. Seed the database by running command "npm run seed-db"
+    - Run the script using commands "npm run server" - this will start the app.
+2. ProductComponent-master
```

Comment:
I'm a little confused with these instructions. How come you're including instructions to run the other components?